### ISSUE TITLE
Error 404

### ISSUE THREAD

**Issue by JAremko** [jcs090218/setup-emacs-windows#56]
I'm getting: 
```
Run jcs090218/setup-emacs-windows@master
Installing Emacs
  Error: Unexpected HTTP response: 404
```

**Comment by JAremko** [comment]
Actually it's gone now. Hmmm. :thinking:

**Comment by jcs090218** [comment]
I think you are referring to the `snapshot`. This should be fixed by 8913223439fadbc538298f20a3d4f2cf451d90e9.
